app/components/search.tsx

```diff
@@ -171,13 +194,86 @@ const ResultItem = ({
       key={title}
       title={tooltip}
       onClick={() => onSelect(pageid, title)}
-      onMouseDown={() => setHide(false)}
+      onMouseDown={() => setHide && setHide(false)}
     >
       {title}
     </button>
   )
 }
 
+const ShowMoreSuggestions = ({
+  question,
+  relatedQuestions,
+  openQuestionTitles,
+  onClose,
+  onSelect,
+}: {
+  question: string
+  relatedQuestions: string[]
+  openQuestionTitles: Props['openQuestionTitles']
+  onSelect: Props['onSelect']
+  onClose: (e: any) => void
+}) => {
+  const title = `Does any of the following questions match your question?`
```

Comment:
@plexish @ccstan99 what should this be?

Reply:
How about "Here are some questions we're still answering. Do any of these match what your question?"

Reply:
or maybe
> Here are some questions we're still answering. Are any of these what you're asking?

to avoid the double use of "question"